README.md

```diff
@@ -50,6 +50,26 @@ Divvy allows users to enter expenses as they are incurred, and calculates the be
 
 ### Installing Dependencies (tbd)
 
+### Setting Up passport-facebook
+
+1. Sign up to be a [facebook developer](https://developers.facebook.com/).
+2. Click the "Add a New App" on your [apps page](https://developers.facebook.com/apps/).
```

Comment:
The button is called "Create a New App" (the subsequent pop-up is titled "Add a New App")